src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
+use super::GLYPH_WIDTH;
+use super::HORIZONTAL_MARGIN;
+use super::VERTICAL_MARGIN;
+use super::TEXT_SIZE;
```

Comment:
Variables `SEPARATOR_MARGIN` and `ICON_MARGIN` do not work as they should. Setting `SEPARATOR_MARGIN` to `0`, makes no margin on the left of the icon. Either the name `ICON_MARGIN` is misleading and is by design applied only to the right side, or this is not working properly:
    ![image](https://user-images.githubusercontent.com/1623053/89510149-7f789600-d7d0-11ea-8129-97efa56eee4b.png)

Reply:
It should have been named `ICON_RIGHT_MARGIN`. I am fixing it